css/amp.css

```diff
@@ -323,6 +323,37 @@ i-amp-scroll-container.amp-active {
   visibility: hidden;
 }
 
+.-amp-loader-line {
+  /* TODO(zhouyx): Confirm on final height, color */
+  position: absolute;
+  top:0;
+  left: 0;
+  right: 0;
+  height: 2px !important;
```

Comment:
can we avoid `!important`? probably with a more accurate selector: `div.-amp-loader-line`?